src/components/SizeContainer/SizeContainer.js

```diff
@@ -0,0 +1,51 @@
+import { Button, makeStyles } from '@material-ui/core';
+import React from 'react';
+import { SIZES } from '../../utility';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    marginLeft: '5px',
+  },
+  sizeBtn: {
+    borderRadius: '50%',
+    minWidth: '40px',
+    minHeight: '40px',
+    border: '1px solid #0000001f',
+    margin: '0 5px',
+
+    '&:hover': {
+      borderColor: '#3f51b5',
+      backgroundColor: '#3f51b51f',
+    },
+    '&:focus': {
+      borderColor: '#3f51b5',
+      backgroundColor: '#3f51b51f',
+    },
+  },
+  active: {
+    borderColor: '#3f51b5',
+    backgroundColor: '#3f51b51f',
+  },
+}));
+
+const SizeContainer = ({ value, onClick }) => {
+  const classes = useStyles();
+  return (
+    <div className={classes.root}>
+      {SIZES.map(size => {
+        return (
+          <Button
+            key={size}
+            variant="text"
+            className={`${classes.sizeBtn} ${size === value ? classes.active : ''}`}
+            onClick={() => onClick(size)}
+          >
+            {size}
+          </Button>
```

Comment:
Instead of the Button component, try using Chip

https://material-ui.com/components/chips/#chip-playground

Something like this:
```
<Chip variant="outlined" color="primary" avatar={<Avatar>M</Avatar>} />
```

And vary the `color` prop based on whether that option is selected or not.

Reply:
There are many factors being considered while using the button component here like accessibility, this is a user action area and we cannot use read-specific components  (Chip is majorly used to read data)
I cant take this into consideration.

Reply:
That is a good point, however, this component can be used as clickable with role attribute.

Check the example here:
https://material-ui.com/components/chips/#chip-2

Reason for the suggestion being, to write less code by using what's already available.